portfolio/src/main/webapp/style.css

```diff
@@ -1,30 +1,46 @@
+/* Main Content of Site */
 #content {
   margin-left: auto;
   margin-right: auto;
-  margin-top: 100px;
+  margin-top: 120px;
   max-width: 850px;
+  line-height: 26px;
+  font-size: 16px;
+  padding-bottom: 50px;
 }
 
-#greeting-container {
+/* Site Sections */
+.section {
+    padding-top: 20px;
+}
+
+/* Arlo Talks Sub-Section */
+#arlo-talks-container {
   margin-top: 20px;
+  bottom: 0;
 }
 
+/* Images Sizing */
 img {
     height: auto;
     max-width: 100%;
 }
 
+/* Body Styles */
 body {
     background-color: rgb(253, 245, 230, 0.7);
     font-family: sans-serif;
 }
 
-p {
-    line-height: 26px;
-    font-size: 16px;
+/* List Items Spacing Styling */
+li {
+    margin-bottom: 5px;
 }
 
+/* Sticky Navbar */
```

Comment:
I like how this navbar looks on the page - nice!

Reply:
Thanks!!